ERCS/erc-7087.md

```diff
@@ -0,0 +1,163 @@
+---
+eip: 7087
+title: New features for Web3 URL in Auto Mode
+description: Add features to Web3 URL in auto mode
```

Comment:
Try not to restate your title in the description. The description should _expand_ on the title.